tools/mock_node/benches/sync.rs

```diff
@@ -0,0 +1,127 @@
+#[macro_use]
```

Comment:
let's also add a README.md file to this benches directory (with info on what's inside, how to run these tests and how to run them against the 'custom' directory)

Reply:
added